src/utils.ts

```diff
@@ -164,3 +164,159 @@ export const proposalSchema = {
   },
   required: ["signatures", "targets", "params", "values", "meta", "type"],
 };
+interface AssetConfig {
+  name: string;
+  address: string;
+  feed: string;
+}
+
+const ASSETS: AssetConfig[] = [
```

Comment:
I would rename it to something like `BNB_MAINNET_ASSETS`, to avoid confusions

Reply:
I think BNB is the right name to use: https://www.binance.com/en/blog/ecosystem/introducing-bnb-chain-the-evolution-of-binance-smart-chain-421499824684903436